### DISCUSSION TITLE
Highlight colors and more text formatting.

### DISCUSSION THREAD

**Discussion by Tharon336** [outline/outline#6693] [Self Hosting]
Hi! I am currently self-hosting this project and I am bit frustrated by the minimal text formatting options available. It's usable, I have no BIG complaints besides the highlight option only being yellow. I have a lot of scenarios where I find myself needing more ways to separate/distinguish information in a single 'document' so having at least more options for highlight colors would help a lot. 

Thanks in advance!

**Comment by tommoor** [comment]
I would generally recommend using the notice blocks for this purpose rather than colored text.

If you have an example where colored highlights are particularly useful that would be great.

**Reply by Va-Bank** [reply]
Here's an example of my notes in Nimbus. I think changing font color should be a basic element. And here's an example of how it's done with markers in Outline. It looks really bad. 
<img width="1087" height="713" alt="1" src="https://github.com/user-attachments/assets/780c96db-75a3-4822-a821-2ecbe9b5abc5" />
<img width="933" height="861" alt="outline" src="https://github.com/user-attachments/assets/b37a227f-879d-48ec-a5de-9289d8d6a419" />
Note: If anything, the data in the screenshots is not real.

**Comment by linksilva** [comment]
Colored text it's a nice feature to have as well. I rather prefer using colored text then highlights specially when distinguishing between headers, as their sizes look similar in cases where I have multiple H3 and H4 on the same document. Yes, there is the index but I think they look better for readability if I can change their color.

**Comment by Va-Bank** [comment]
What's really missing is a colored font. Not highlighting, but changing the color of the font itself.

**Comment by willlogs** [comment]
Does colored text have its own request? If not we should open one it should be pretty easy to build.

**Reply by Va-Bank** [reply]
If you know how to make such a request, please do so.

**Reply by willlogs** [reply]
> If you know how to make such a request, please do so.

I'll look into it to see if I can create a pull request if it's actually as easy as I thought it was.
